### PR TITLE
New Dashboard scaffolding

### PR DESCRIPTION
**Pull request by jesusbotella**
This PR lays the foundation stone for New Dashboard™️.
The scaffolding was generated by Vue CLI v2, and shaped to meet our project requirements.

- **Folder Structure**
   - As other projects do, we put all source files under main directory which is `lib/assets/javascripts/new-dashboard`.
   - Source files are splitted by `components`, `pages`, `router` and `store`.
      Separating different concerns.

- **Framework**
   - We chose Vue as our development framework reusing Backbone components via Vue component wrappers.
  
- **Testing**
   - Unit testing is handled via Jest, which runs all test files inside `lib/assets/test/spec/new-dashboard/unit`.
   - It includes Code Coverage, which is handy to keep test tracking from the beginning.
   screen shot 2018-10-17 at 17 05 58

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -26,3 +27,4 @@
 /spec
 /vendor
 /coverage
+/node_modules/
```
Comment (rjimenezda) [inline .eslintignore]: I really hope this is not needed, or we were wasting a lot of energy before :grimacing:

File: lib/assets/javascripts/new-dashboard/.babelrc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "presets": [
+    ["env", {
+      "modules": false,
+      "targets": {
+        "browsers": ["> 1%", "last 2 versions", "not ie <= 8"]
```
Comment (rjimenezda) [inline lib/assets/javascripts/new-dashboard/.babelrc]: I'm not sure, but we probably can get away with not ie <= 11? @ivanmalagon

File: lib/assets/test/spec/new-dashboard/.babelrc [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "presets": [
+    ["env", {
+      "modules": false,
+      "targets": {
+        "browsers": ["> 1%", "last 2 versions", "not ie <= 8"]
```
Comment (rjimenezda) [inline lib/assets/test/spec/new-dashboard/.babelrc]: Same as above

File: .eslintignore (hunk 1 above)
Comment (jesusbotella) [reply .eslintignore]: Yes, it was ignoring /node_modules, so this is not needed 😂.

**Pull request merged**